libjpeg-turbo/idf_component.yml

```diff
@@ -0,0 +1,6 @@
+
+version: "1.1.0~1"
```

Comment:
Let's set the version to match the upstream version of libjpeg-turob we are using.

Reply:
By the way, you seem to be using some commit of libjpeg-turbo from their master branch. There are release tags in libjpeg-turbo repo, we use the release commits unless there is a strong reason not to.